doc/source/Windows.rst

```diff
@@ -0,0 +1,116 @@
+Windows Tutorial 
+================
+
+This guide gives you a brief introduction to how volatility3 works and some demonstration on suite of plugins available from
+
+Acquiring memory
+----------------
+
+Volatility does not provide the ability to acquire memory. 
+
+.. tip:: You could use `WinPmem <https://github.com/Velocidex/WinPmem/releases/latest>`_ for collecting windows memory dump.
+
+Listing Plugins
+---------------
+
+Following are the sample of linux plugins available for volatility3. More plugins will be available on future releases.
+For plugin requests, Please create an issue with description of the plugin.
+
+.. code-block:: shell-session
+
+    $ python3 vol.py --help | grep windows | head -n 5
+        windows.bigpools.BigPools
+        windows.cmdline.CmdLine
+        windows.crashinfo.Crashinfo
+        windows.dlllist.DllList
+                            Lists the loaded modules in a particular windows
+
+.. note:: Here the the command is piped to grep and head in-order to give you sample list of windows plugins.
+
+Using plugins
+-------------
+
+The following is the syntax to run volatility tool.
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f <path to memory image> plugin_name plugin_option
+
+
+Example
+-------
+
+Example 1
+~~~~~~~~~
+
+In this example we will be using memory dump from PragyanCTF'22. 
+We will limit the discussion to memory forensics with volatility3 and not extend to other parts of the challenges. 
+
+In windows memory forensics using volatility3, most of the times we do not require creating a ISF file. 
```

Comment:
This isn't quite accurate.  We always require an ISF file, but often volatility will determine which one we need and generate it from debugging information available from microsoft automatically.

Reply:
I meant it works most of the time that we don't have to create a ISF file manually like Linux